llvm/include/llvm/IR/Attributes.td

```diff
@@ -285,6 +285,9 @@ def SanitizeHWAddress : EnumAttr<"sanitize_hwaddress", [FnAttr]>;
 /// MemTagSanitizer is on.
 def SanitizeMemTag : EnumAttr<"sanitize_memtag", [FnAttr]>;
 
+/// NumericalStabilitySanitizer is on.
+def SanitizeNumericalStability : EnumAttr<"sanitize_numericalstability", [FnAttr]>;
```

Comment:
This should be in the other patch

Reply:
yeah, it's already included into the first patch (https://github.com/llvm/llvm-project/pull/85916), duplicated here to unblock myself (Clang needs this in clang/lib/CodeGen/CGDeclCXX.cpp) (otherwise this PR would have a hard dependency on the first one)